src/pages/api/preview.js

```diff
@@ -0,0 +1,36 @@
+import wp from "@/promisetracker/lib/wp";
+
+export default async function preview(req, res) {
+  const {
+    postType,
+    postId,
+    revisionId,
+    _wpnonce: nonce,
+    _thumbnail_id: thumbnailId,
```

Comment:
1. What is this for?
2. I don't see it from [here](https://github.com/CodeForAfrica/HURUmap-Dashboard/blob/e21644d3040e972902fe3be418f2fd9153b9329c/wp-content/themes/hurumap/functions.php#L237)

Reply:
it's attached by WordPress if a featured image is set

Reply:
Yes but where does it come from? I don't see us passing it in from [here](https://github.com/CodeForAfrica/HURUmap-Dashboard/blob/e21644d3040e972902fe3be418f2fd9153b9329c/wp-content/themes/hurumap/functions.php#L237)

Reply:
I think another hook sets the thumbnail after that particular hook has run. I couldn't find it in the documentation, but that was the behavior. So there was no need to get and set the preview Id.

Reply:
Didn't know about [`post_format`](https://wordpress.org/support/article/post-formats/) which gets included as well (if available of course).